### PR TITLE
Chore: Added Meshery Adapter for Cilium Service Mesh in LifeCycle list

### LINKED ISSUE #2418: Add Meshery Adapter for Cilium Service Mesh as an Alpha adapter
**Current Behavior**

**Desired Behavior**
- [ ] Add to the list on https://layer5.io/service-mesh-management/meshery

---
**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). Fill-in the [Layer5 Community Member Form](https://layer5.io/newcomer) and join the [Layer5 Community Slack](http://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by Abhijay007**
Signed-off-by: Abhijay Jain <Abhijay007j@gmail.com>

**Description**

Added Meshery Adapter for Cilium Service Mesh in LifeCycle list

This PR fixes #2418 

**Notes for Reviewers**

Screenshot : 

image

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: src/sections/Meshery/Features-Col/data/index.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ const data = {
       btnText: "Get Started",
       services: [
         {
-          content: "AWS App Mesh, Citrix Service Mesh, Consul, Istio, Kuma, Linkerd, Network Service Mesh, NGINX Service Mesh, Octarine, Open Service Mesh, Traefik Mesh",
+          content: "AWS App Mesh, Citrix Service Mesh, Consul, Istio, Kuma, Linkerd, Network Service Mesh, NGINX Service Mesh, Octarine, Open Service Mesh, Traefik Mesh, Cilium Service Mesh",
```
Comment (leecalcote) [inline src/sections/Meshery/Features-Col/data/index.js:10]: The rest of the list is alphabetical. Please fix.

Comment (Abhijay007) [reply src/sections/Meshery/Features-Col/data/index.js:10]: ok

Comment (Abhijay007) [reply src/sections/Meshery/Features-Col/data/index.js]: it is fixed now !! 

image

**Comment by l5io** [comment]
🚀 Preview for commit 70f1bc919c1deb6aa7ce628e67d7e1ca261f89b4 at: https://61bbb9e25c6992167a1bacb0--layer5ng.netlify.app

**Review by leecalcote — APPROVED** [review]
Looking good. 👍

**Pull request merged**